CONTRIBUTING.md

```diff
@@ -39,3 +44,5 @@ In order to tag the repository, use the following command:
 ```sh
 npm version <patch|minor|major>
```

Comment:
isn't this outdated? we use `pnpm changeset` in this repo right?

Reply:
no, we still use npm version to tag